Foundation/NSURLSession/NSURLSessionTask.swift

```diff
@@ -543,13 +543,14 @@ extension _ProtocolClient : URLProtocolClient {
             task.state = .completed
             session.taskRegistry.remove(task)
         case .dataCompletionHandler(let completion):
-            let data = Data()
```

Comment:
We will need to create this data if `protocol.properties[.responseData]` is nil.